x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/signals/query_signals_route.test.ts

```diff
@@ -62,17 +62,13 @@ describe('query for signal', () => {
     test('returns 200 when using aggs and query together', async () => {
       const allTogether = getSignalsQueryRequest();
       allTogether.payload = { ...typicalSignalsQueryAggs(), ...typicalSignalsQuery() };
-      elasticsearch.getCluster = jest.fn(() => ({
-        callWithRequest: jest.fn(
-          (_req, _type: string, queryBody: { index: string; body: object }) => {
-            expect(queryBody.body).toMatchObject({
-              ...typicalSignalsQueryAggs(),
-              ...typicalSignalsQuery(),
-            });
-            return true;
-          }
-        ),
-      }));
+      clients.clusterClient.callAsCurrentUser.mockImplementation((endpoint, params) => {
+        expect(params!.body).toMatchObject({
+          ...typicalSignalsQueryAggs(),
+          ...typicalSignalsQuery(),
+        });
+        return Promise.resolve(true);
```

Comment:
Same issue here with these tests, they are not being reached.

I tested it by changing the code to be:

```ts
      clients.clusterClient.callAsCurrentUser.mockImplementation((endpoint, params) => {
        expect(true).toEqual(false);
        expect(params!.body).not.toEqual({
          ...typicalSignalsQueryAggs(),
          ...typicalSignalsQuery(),
        });
        return Promise.resolve(true);
      });
```

and did not see any errors which makes me think either the originals at some point stopped being reached or these news ones are not being reached. In either case we want to fix them to work (hopefully)

Reply:
56fc3f5

Reply:
56fc3f5292a0893138b4e98af7116b02c671e057